Sources/Controllers/Map/Widgets/OADestinationsLineWidget.mm

```diff
@@ -301,11 +314,15 @@ - (double) getStrokeWidth
 - (UIImage *)getArrowImage:(UIImage *)fgImage inImage:(UIImage *)bgImage withShadow:(UIImage *)shadow
 {
     UIGraphicsImageRenderer *renderer = [[UIGraphicsImageRenderer alloc] initWithSize:bgImage.size];
-    return [renderer imageWithActions:^(UIGraphicsImageRendererContext * _Nonnull rendererContext) {
-        [shadow drawInRect:CGRectMake(0.0, 0.0, shadow.size.width, shadow.size.height)];
-        [bgImage drawInRect:CGRectMake(0.0, 0.0, bgImage.size.width, bgImage.size.height)];
-        [fgImage drawInRect:CGRectMake(0.0, 0.0, fgImage.size.width, fgImage.size.height)];
-    }];
+    if (renderer)
```

Comment:
When will renderer be nil? initWithSize returns instancetype, not - (nullable instancetype)